.github/workflows/chart-releaser.yaml

```diff
@@ -8,17 +8,28 @@ jobs:
   release:
     permissions:
       contents: write
+      pages: write
+      id-token: write
     runs-on: ubuntu-latest
+    environment:
+      name: github-pages
+      url: ${{ steps.deployment.outputs.page_url }}
     steps:
+      - uses: actions/create-github-app-token@v1
+        id: app-token
+        with:
+          app-id: ${{ secrets.APP_ID }}
+          private-key: ${{ secrets.AUTOMATION_KEY }}
+
       - name: Checkout
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
 
       - name: Configure Git
         run: |
-          git config user.name "$GITHUB_ACTOR"
-          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+          git config user.name "opentdf-automation[bot]"
+          git config user.email "123456789+opentdf-automation[bot]@users.noreply.github.com"
```

Comment:
Why are you putting the number prefix in the email here?

Reply:
going off of this knowledge base here https://github.com/orgs/community/discussions/24664#discussioncomment-3244950

Its old so it might be outdated.

Reply:
yeah thats how i got it to work

Reply:
Reading the linked docs the number should be the app user id, `123456789` was just used as an example.

Reply:
I did not update that id to match the github app id so good catch here.